content/concepts/dolthub/api.md

```diff
@@ -313,3 +313,166 @@ res = requests.get(
 with open(local_file, 'wb') as file:
     file.write(res.content)
 ```
+
+## Database API
```

Comment:
Could use a little explanation here about what the Database API is and how it's different from the SQL API